script.js

```diff
@@ -1,16 +1,53 @@
 // Write your JavaScript code here!
 
-window.addEventListener("load", function() {
+//const { pickPlanet } = require("./scriptHelper");
 
-   let listedPlanets;
-   // Set listedPlanetsResponse equal to the value returned by calling myFetch()
-   let listedPlanetsResponse;
-   listedPlanetsResponse.then(function (result) {
-       listedPlanets = result;
-       console.log(listedPlanets);
-   }).then(function () {
-       console.log(listedPlanets);
-       // Below this comment call the appropriate helper functions to pick a planet fom the list of planets and add that information to your destination.
-   })
-   
-});
\ No newline at end of file
+window.addEventListener("load", function () {
+  let listedPlanets;
+  // Set listedPlanetsResponse equal to the value returned by calling myFetch()
+  let listedPlanetsResponse = myFetch();
+  listedPlanetsResponse
+    .then(function (result) {
+      listedPlanets = result;
+      console.log(listedPlanets);
+    })
+    .then(function () {
+      console.log(listedPlanets);
+      // Below this comment call the appropriate helper functions to pick a planet fom the list of planets and add that information to your destination.
+      let planetVar = pickPlanet(listedPlanets);
+      console.log(planetVar);
+
+      addDestinationInfo(
+        document,
+        planetVar.name,
+        planetVar.diameter,
+        planetVar.star,
+        planetVar.distance,
+        planetVar.moons,
+        planetVar.image
+      );
+    });
+
+  let form = document.querySelector("form");
```

Comment:
Grab the `faultyItems` element in `script.js` and set its visibility to `hidden` inside your load event listener (above your submit event listener on line 32)